src/components/TrapFocus/TrapFocus.tsx

```diff
@@ -72,21 +73,24 @@ export class TrapFocus extends React.PureComponent<TrapFocusProps, State> {
     const {focusTrapWrapper} = this;
     const {trapping = true} = this.props;
 
-    if (relatedTarget == null || trapping === false) {
+    if (trapping === false) {
       return;
     }
 
     if (
       focusTrapWrapper &&
       !focusTrapWrapper.contains(relatedTarget as HTMLElement) &&
-      !closest(relatedTarget as HTMLElement, '[data-polaris-overlay]')
+      (relatedTarget
```

Comment:
This ternary is doing the same thing as the previous check `relatedTarget == null`.

Reply:
`!relatedTarget || !closest(relatedTarget as HTMLElement, '[data-polaris-overlay]')`

Would this work as well rather than the nested conditionals?

Reply:
It's lovely 🎉